src/ampycloud/data.py

```diff
@@ -992,6 +1010,28 @@ def layers(self) -> pd.DataFrame:
         identified by the layering algorithm. """
         return self._layers
 
+    @property
+    def high_clouds_detected(self) -> bool:
+        """ Returns whether high clouds were detected in the data.
```

Comment:
How should this be interpreted exactly? 
No cloud hits above the MSA? Above the MSA + buffer? (before running ampycloud)
No cloud layer base above the MSA? (after running ampycloud)
We should write the docstring to clarify.

Reply:
see suggestion above.